### PR TITLE
Update UG with images and allow Edit Command to show details in Display Panel

### LINKED ISSUE #151: Find command wrong error message
Screenshot 2024-10-31 at 9 26 04 PM

Parameters explainer not needed

### PR DESCRIPTION
**Pull request by LemonDrew**
closes #151 and improve quality of test cases

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -157,6 +157,13 @@ Format: `view n/NAME j/JOB`
   `Software Engineer`.
 * Only full words will be matched e.g. `Han` will not match `Hans` and `software engineers` will not match
   `Software Engineer`.
+* Details of the person being viewed will be displayed in the display panel
```
Comment (SQ77) [inline docs/UserGuide.md:160]: Add a full stop.

**Review by SQ77 — COMMENTED** [review]
Should use the updated image from the latest UI for the example.

File: docs/images/Ui.png
Comment (SQ77) [inline docs/images/Ui.png:1]: Possible to make the image similar size as the previous image (mockup)?

File: src/test/java/seedu/address/logic/parser/AddressBookParserTest.java [hunk 2]
```diff
@@ -198,24 +198,49 @@ public void parseCommand_withExtraWhitespace() throws Exception {
         assertEquals(expectedCommand.job, command.job);
     }
 
+    // Invalid Input (Name not specified in hire command)
     @Test
-    public void parseCommand_invalidHireCommandFormat_throwsParseException() {
+    public void parseCommand_invalidHireCommandFormatMissingName_throwsParseException() {
         assertThrows(ParseException.class, String.format(MESSAGE_INVALID_COMMAND_FORMAT, HireCommand.MESSAGE_USAGE), ()
                 -> parser.parseCommand("hire n/Amy Bee"));
```
Comment (SQ77) [inline src/test/java/seedu/address/logic/parser/AddressBookParserTest.java:205]: Should this be job not specified?

File: src/test/java/seedu/address/logic/parser/AddressBookParserTest.java [hunk 3]
```diff
@@ -198,24 +198,49 @@ public void parseCommand_withExtraWhitespace() throws Exception {
...
     }
 
+    // Invalid Input (Job not specified in hire command)
     @Test
-    public void parseCommand_invalidRejectCommandFormat_throwsParseException() {
+    public void parseCommand_invalidHireCommandFormatMissingJob_throwsParseException() {
+        assertThrows(ParseException.class, String.format(MESSAGE_INVALID_COMMAND_FORMAT, HireCommand.MESSAGE_USAGE), ()
+                -> parser.parseCommand("hire j/Software Engineer"));
+    }
```
Comment (SQ77) [inline src/test/java/seedu/address/logic/parser/AddressBookParserTest.java:213]: Should this be name not specified?

**Review by SQ77 — APPROVED** [review]
LGTM!

**Review by bigismols — APPROVED** [review]
LGTM

**Review by bigismols — APPROVED** [review]
LGTM!

**Pull request merged**